starter/source/materials/mat/mat069/hm_read_mat69.F

```diff
@@ -137,8 +137,15 @@ SUBROUTINE HM_READ_MAT69(UPARAM ,MAXUPARAM,NUPARAM  ,ISRATE   , IMATVIS,
       IF (ICHECK == 0)     ICHECK = -3
       IF (NSTART == 0)     NSTART = 100
       IF (ERRTOL == ZERO ) ERRTOL = FIVEEM3
-      IF (NMUAL == 0)      NMUAL = 2
-      IF (FSCAL == ZERO)   FSCAL = ONE*FSCAL_UNIT
+      IF (NMUAL == 0     ) NMUAL = 2
+      IF (FSCAL == ZERO  ) FSCAL = ONE*FSCAL_UNIT
+      IF(LAWID == 0      ) LAWID = 3
```

Comment:
Hi @SaidMamouri 
can you correct indentation , to have homogeneous code ?

Thanks